data/sql_updates/create_candidates_view.sql

```diff
@@ -29,8 +29,7 @@ select
     max(candprops.cand_nm) as name
 from dimcand
     left join (select distinct on (cand_sk) cand_sk, election_yr, cand_status, ici_code from dimcandstatusici order by cand_sk, election_yr desc) csi_recent using (cand_sk)
-    left join dimcandstatusici csi_all using (cand_sk)
-    inner join dimcandoffice co on co.cand_sk = dimcand.cand_sk and (csi_recent.election_yr is null or co.cand_election_yr = csi_recent.election_yr)  -- only joined to get to dimoffice
+    inner join dimcandoffice co on co.cand_sk = dimcand.cand_sk --and (csi_recent.election_yr is null or co.cand_election_yr = csi_recent.election_yr)
```

Comment:
Very similar to the last question, but how come we're filtering on `csi_recent.election_yr is null or co.cand_election_yr = csi.election_yr` in the detail view but not in the list view?